README.md

```diff
@@ -1,3 +1,72 @@
-# dockercomposehost
+# aws-dockercomposehost
 
 Terraform module for creating AWS EC2 instances with Docker and Docker Compose installed.
+
+## Example
+
+docker-compose.yml:
+```yaml
+version: '3'
+
+services:
+  hello:
+    image: nginxdemos/hello
+    ports:
+      - "80:80"
+```
+
+main.tf:
+```hcl
+module "aws-dockercomposehost" {
+  source = "git@github.com:/buildo/terraform-aws-dockercomposehost.git?ref=9-terraform_0_13"
```

Comment:
Let's avoid referencing this branch, it will become obsolete as soon as we merge into master 😄

Reply:
Whoops, my fault